README.md

```diff
@@ -8,7 +8,7 @@ If you want to contribute to this project, feel free to fork this repository and
 **As of 09/01/20, this project is under no active development by the horion team anymore. We will still review PRs.**
 
 [![Discord](https://img.shields.io/discord/732833913705201736?label=chat&logo=discord&logoColor=white&style=for-the-badge)](https://discord.gg/WPe3WeNtgv)
-[![Download the injector](https://img.shields.io/badge/download-injector-brightgreen?style=for-the-badge "Download the injector")](https://horionbeta.club/injector "Download the injector")
+[![Download the injector](https://img.shields.io/badge/download-injector-brightgreen?style=for-the-badge "Download the injector")](https://github.com/horionclient/Horion/raw/master/Injector/HorionRelease.zip "Download the injector") <!--https://horionbeta.club/injector-->
```

Comment:
This link can only work if this is merged.

Reply:
Oof this link doesnt work at all

Reply:
> Oof this link doesnt work at all

Did you read my comment?

Reply:
when will they merge lmao